oak/proto/manager.proto

```diff
@@ -33,8 +34,11 @@ message ApplicationConfiguration {
   // Indication of what configuration the initial Node should run.  Must identify a
   // NodeConfiguration entry that holds a WebAssemblyConfiguration object.
   string initial_node = 2;
```

Comment:
I think a better name for this now would be `initial_config_name`, and `initial_entrypoint_name` for the other field, and also generally converge towards `config_name` and `entyrpoint_name` elsewhere too (there are a few inconsistencies between this and `oak_abi` and `oak` SDK currently). WDYT?